src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java

```diff
@@ -24,12 +24,14 @@
      * Type B is a class that extends {@link AbstractBuildingWorker.View}.
      */
     protected final B          building;
-    private final   SwitchView switchView;
     private final   Label      title;
-    private final   Button     buttonPrevPage;
-    private final   Button     buttonNextPage;
     private final   Button     buttonBuild;
     private final   Button     buttonRepair;
```

Comment:
### Errors:
- Type of field 'buttonRepair' is concrete class Button 

### Warnings:
- Instance field name buttonRepair doesn't match regex 'm_[a-z][A-Za-z\d]*'